### DISCUSSION TITLE
Консоль, предупреждение:  файлы хостов в режиме чтения

### DISCUSSION THREAD

**Discussion by nialone** [bol-van/zapret#969] [Q&A]
Стояла v67, поставил v69.8. Если через консоль (telnet) роутера перезапустить запрет, то постоянно в нем же сыпется предупреждение НИЖЕ ... пока не закрыть консоль.

`Opkg::Manager: /opt/etc/init.d/rc.unslung: cannot access hostlist file '/opt/zapret/ipset/lovetube.txt'. in-memory content remains unchanged.`

И так по трём: `lovetube.txt`,`zapret-hosts-user.txt`, `zapret-hosts-user-exclude.txt`
Откатился обратся на v67, тишина. Не знаю, это фича или баг. Соррян за флуд. )

**Comment by bol-van** [comment]
Инсталятор сносит юзерские файлы из /opt/zapret, поэтому их там не следует хранить
Сообщение означает, что в момент запуска хостлист был, а потом его не стало

Либо были выставлены слишком рестриктивные права на /opt
chmod 700, например.
После сброса привилегий доступа нет
